helix-contract/contracts/mapping-token/v2/erc20-mapping-protocol/WETH.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.4.18;
+
+contract WETH9 {
+    string public name     = "Wrapped Ether";
+    string public symbol   = "WETH";
```

Comment:
Should we rename it to WCRAB / WRING ?

Reply:
This is for testing, the official contract needs to consider this Name and Decimals